_includes/navigation.html

```diff
@@ -2,6 +2,7 @@
   <div class="c-navigation__container u-container">
     <a class="c-navigation__item {% if page.url == '/' %}is-active{% endif %}" href="{{ "/" | prepend: site.baseurl }}">Home</a>
     <a class="c-navigation__item {% if page.url == '/articles/' %}is-active{% endif %}" href="{{ "/articles/" | prepend: site.baseurl }}">Articles</a>
+    <a class="c-navigation__item {% if page.url == '/tags/' %}is-active{% endif %}" href="{{ "/tags/" | prepend: site.baseurl }}">Tags</a>
```

Comment:
I would almost prefer this not to be a top-header link on its own, rather a link at the top of the Articles page. I'll leave that up for discussion though.